api/api.proto

```diff
@@ -3405,6 +3419,19 @@ message SdkNodeVolumeUsageByNodeResponse {
   VolumeUsageByNode volume_usage_info = 1;
 }
 
+// Defines request to trigger RelaxedReclaim purge
+// for a given node
+message SdkNodeRelaxedReclaimPurgeRequest {
+  // Id of the node to trigger the purge
+  string node_id = 1;
+}
+
+// Defines response containing status of the trigger
+message SdkNodeRelaxedReclaimPurgeResponse {
+  // status returns true on successful purge
+  RelaxedReclaimPurge status = 1;
```

Comment:
Do we need another message here?  It can be bool here rather than the message for status ?